articles/storage/blobs/storage-manage-find-blobs.md

```diff
@@ -285,12 +285,7 @@ Blob index tags are only available on General Purpose v2 (GPv2) accounts with hi
 
 Index tags aren't supported on Premium storage accounts. For more information about storage accounts, see [Azure storage account overview](../common/storage-account-overview.md).
 
-In public preview, blob index tags are only available in the following regions:
-
-- Canada Central
-- Canada East
-- France Central
-- France South
+Blob index tags are currently available in all **public regions**
```

Comment:
Please remove the bolding on "public regions" and add a period to the end of the sentence.